src/nxdoc/web-ui/web-ui-overview.md

```diff
@@ -106,41 +105,39 @@ version_override:
 
-It leverages the web as a platform and allows enhanced productivity with direct access to last visited documents, collections, easy browsing between previous/next document, full-screen suggester and several other features built to provide a rich user experience. Here are the most important topics on getting started with Web UI, customizing it and building new web apps using Nuxeo Elements:
+It leverages the web as a platform and allows enhanced productivity with direct access to last visited documents, collections, easy browsing between previous/next document, full screen suggester and several other features built to provide a rich user experience. Here are the most important topics on getting started with Web UI, customizing it and building new web apps using Nuxeo Elements:
+
+<div class="row"><div class="column medium-4">{{#> panel type='primary'}}
 
-<div class="row">
-<div class="column medium-4" align="center">
-{{#> panel type='primary'}}
 [Polymer Guide]({{page page='polymer-guide'}})
-{{/panel}}
-</div>
-<div class="column medium-4" align="center">
-{{#> panel type='primary'}}
+
+{{/panel}}</div><div class="column medium-4">{{#> panel type='primary'}}
+
 [Nuxeo Elements]({{page page='nuxeo-elements'}})
-{{/panel}}
-</div>
-<div class="column medium-4" align="center">
-{{#> panel type='primary'}}
+
+{{/panel}}</div><div class="column medium-4">{{#> panel type='primary'}}
+
 [Customization]({{page page='web-ui-customization'}})
-{{/panel}}
-</div>
-</div>
+
+{{/panel}}</div></div>
 
 {{#> callout type='info' heading='Nuxeo University'}}
-Watch the related courses on Nuxeo University:</br>
-[Course on the Web UI Stack](https://university.nuxeo.com/learn/public/course/view/elearning/76/nuxeo-web-ui-stack-introduction).</br>
-[Course to Start with Web UI](https://university.nuxeo.com/learn/public/course/view/elearning/92/getting-started-with-nuxeo-web-ui).
+Watch the related courses on Nuxeo University
+- [Course on the Web UI Stack](https://university.nuxeo.com/learn/public/course/view/elearning/76/nuxeo-web-ui-stack-introduction).
+- [Course to Start with Web UI](https://university.nuxeo.com/learn/public/course/view/elearning/92/getting-started-with-nuxeo-web-ui).
 ![]({{file name='university-webui.png' page='nxdoc/university'}} ?w=450,border=true)
 {{/callout}}
 
 ## Requirements
 
 {{! multiexcerpt name='webui-requirements'}}
-Nuxeo Web UI and Nuxeo Web UI Elements framework make use of the recent W3C standard Web Components and Google Polymer 2.0 framework (legacy mode). The Google Polymer framework comes with polyfills for Web Components, a library that extends the support of web components standard to more browsers.
+Nuxeo Web UI and Nuxeo Web UI Elements framework make use of the recent W3C standard Web Components and Google Polymer 3.0 framework (legacy mode). The Google Polymer framework comes with polyfills for Web Components, a library that extends the support of web components standard to more browsers.
```

Comment:
We can scrap the legacy from here. Elements are on polymer 3 with class based syntax, and Web UI is on polymer 3 as well, were some elements are use classed based syntax.